app/uk/gov/hmrc/gform/sharedmodel/DataRetrieve.scala

```diff
@@ -101,6 +101,7 @@ object DataRetrieve {
     case object String extends AttrType
     case object Integer extends AttrType
     case object Date extends AttrType
+    case object BigDecimal extends AttrType
```

Comment:
From the look of it `BigDecimal` behaves exactly as `Integer` (looking at gform-frontend). 

I think we should rename `Integer` (line 102) to a `Number` instead rather than having two types which behaves exactly the same.

Reply:
Yeah okay, that makes sense. Will get on it.

Reply:
Hi @josef-vlach, I've renamed this now and pushed the changes up.